core/trino-main/src/main/java/io/trino/testing/TestingAccessControlManager.java

```diff
@@ -125,9 +125,14 @@
     private BiPredicate<Identity, String> denyIdentityTable = IDENTITY_TABLE_TRUE;
 
     @Inject
+    public TestingAccessControlManager(TransactionManager transactionManager, EventListenerManager eventListenerManager, AccessControlConfig accessControlConfig)
+    {
+        super(transactionManager, eventListenerManager, accessControlConfig, DefaultSystemAccessControl.NAME);
+    }
+
     public TestingAccessControlManager(TransactionManager transactionManager, EventListenerManager eventListenerManager)
```

Comment:
why do you need this constructor?

Reply:
There are ~6 tests where this class is instantiated directly (w/o guice injection), I've decided to leave this "old" constructor to not pollute existing code with new argument that is not required nor have special usage.